network/gossip.go

```diff
@@ -126,7 +126,6 @@ func (s *Server) NewTopic(protoID string, obj proto.Message) (*Topic, error) {
 		topic:   topic,
 		typ:     reflect.TypeOf(obj).Elem(),
 		closeCh: make(chan struct{}),
-		closed:  new(uint64),
```

Comment:
Default value?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1407/commits/09b94cecd1c6f00cc37961536058a8c6d0dbeed6. 

Although the documentation states that by default it is `false`.